src/Compilers/Server/VBCSCompilerTests/BuildClientTests.cs

```diff
@@ -31,14 +32,16 @@ public sealed class ServerTests : BuildClientTests
             private readonly string _pipeName = Guid.NewGuid().ToString("N");
             private readonly BuildPaths _buildPaths;
             private readonly List<ServerData> _serverDataList = new List<ServerData>();
+            private readonly XunitCompilerServerLogger _logger;
```

Comment:
nit: should this be an `ICompilerServerLogger`?

Reply:
In this case the field is unconditionally initialized as `XunitCompilerServerLogger` hence I decided to keep the field type reflective of that.